cros_gralloc/cros_gralloc_driver.cc

```diff
@@ -177,7 +277,10 @@ cros_gralloc_driver::cros_gralloc_driver()
 			continue;
 
 		if (!strcmp(version->name, "virtio_gpu")) {
-			virtio_node_idx = availabe_node;
+			if (virtio_node_idx == -1)
+				virtio_node_idx = availabe_node;
```

Comment:
We are assuming that the first virtio-GPU device would always be virtio-pci device, and the others are virtio-ivshmem devices, right? Why don't we call `drv_virtgpu_is_ivshm()` to check it?

Reply:
The drv_virtgpu_is_ivshm() will call success after the drv init, in this step, the driver is not create/init, couldn't get the the result.